yarn-project/noir-contracts/src/contracts/pending_commitments_contract/src/main.nr

```diff
@@ -23,6 +23,11 @@ contract PendingCommitments {
         note_header::NoteHeader,
         utils as note_utils,
     };
+    use dep::aztec::types::point::Point;
+    use dep::aztec::state_vars::map::Map;
+    use dep::aztec::state_vars::set::Set;
+    use dep::aztec::constants_gen::MAX_NOTE_FIELDS_LENGTH;
+    use dep::aztec::private_call_stack_item::PrivateCallStackItem;
```

Comment:
Why did these extra imports need to be added to this file?

Reply:
Wrongly resolved merge conflict 😬 Thanks for spotting this!